LAKMobile/src/components/JobThumbnail.tsx

```diff
@@ -49,18 +51,42 @@ function StatusIndicator({text, color}: StatusIndicatorProps) {
     )
 }
 
+/**
+ * Helpers
+ */
+const getDisplayImage = (job: JobData): ImageSourcePropType => {
+    const imageId = job.imageIds[0];
+    return imageIdToSource(imageId);
+}
+
+const diffDatesInDays = (start: Date, end: Date) => {
+    let diff = (end.getTime() - start.getTime()) / (1000 * 3600 * 24);
+    return diff < 0 ? 0 : Math.round(diff);
+}
+
 /**
  * Job thumbnail component
+ * Note: Display of the component varies based on whether the owner or an applicant views it which is why different type of props for the two cases
  */
-interface JobThumbnailProps {
-    job: JobData;
-    image: ImageSourcePropType;
-    displayStatus: 'In Progress' | 'Finished' | 'Accepted' | 'Denied' | 'Not Started' | 'Applied',
-    isJobOwner?: boolean;
-    daysAgo?: number;
+
+interface JobThumbnailOwnerViewProps {
+    isJobOwner: true;
+    job: JobOwnerView;
     repostAllowed?: boolean;
 }
-const STATUS_DISPLAY_COLOR = {
+
+interface JobThumbnailApplicantViewProps {
+    isJobOwner: false;
+    job: JobData;
+    appliedDate: Date;
+    applicantStatus: 'Applied' | 'Accepted' | 'Denied';
+}
+
+type JobThumbnailProps = JobThumbnailOwnerViewProps | JobThumbnailApplicantViewProps;
+
+type DisplayStatus = 'In Progress' | 'Finished' | 'Accepted' | 'Denied' | 'Not Started' | 'Applied';
```

Comment:
Not all the display status will be used in one view (owner or applicant). 

Here is the flow of status in respective views
Owner View
Not Started -> In Progress -> Finished

Applicant View
Not Started | Applied | Accepted/Denied | Finished